### PR TITLE
[Docs] Customizing Credit Memo's logo

### PR DESCRIPTION
**Pull request by TheMilek**
| Q               | A
| --------------- | -----
| Branch?         | 1.10 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no 
| Related tickets | [PR](https://github.com/Sylius/RefundPlugin/pull/343)
| License         | MIT

### CODE REVIEW

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 1]
```diff
@@ -21,6 +21,17 @@ The first exemplary customization is to change background color of the heading o
             /* ... */
         </style>
 
+Customizing Credit Memo's logo
+----------------------------------
+
+Credit Memo's logo is by default displayed as Sylius logo.
+Change it by modifying the ``SYLIUS_REFUND_LOGO_FILE`` environment variable.
```
Comment (lchrusciel) [inline docs/cookbook/payments/customizing-credit-memo.rst:28]:
```suggestion
Changing it is done by modifying the ``SYLIUS_REFUND_LOGO_FILE`` environment variable.
```

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 2]
```diff
@@ -21,6 +21,17 @@ The first exemplary customization is to change background color of the heading o
             /* ... */
         </style>
 
+Customizing Credit Memo's logo
+----------------------------------
```
Comment (CoderMaggie) [inline docs/cookbook/payments/customizing-credit-memo.rst:25]:
```suggestion
Customizing Credit Memo's logo
------------------------------
```

**Review by CoderMaggie — APPROVED** [review]

File: docs/cookbook/payments/customizing-credit-memo.rst (hunk 1 above)
Comment (CoderMaggie) [reply docs/cookbook/payments/customizing-credit-memo.rst]: why? :D much more formal tone than in our docs usually.

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 3]
```diff
@@ -21,6 +21,17 @@ The first exemplary customization is to change background color of the heading o
...
+
+Credit Memo's logo is by default displayed as Sylius logo.
+Changing it is done by modifying the ``SYLIUS_REFUND_LOGO_FILE`` environment variable.
+You can achieve that by overwriting it's path in your ``.env`` file, like in the example below:
+
+    .. code-block:: text
+
+        SYLIUS_REFUND_LOGO_FILE=/Users/Sylius/logo_directory/your_logo.png
```
Comment (GSadee) [inline docs/cookbook/payments/customizing-credit-memo.rst:33]: Similar to https://github.com/Sylius/Sylius/pull/13421#pullrequestreview-843431184, you should put a custom logo in the project directory and then use it with the proper path.

**Review by GSadee — CHANGES_REQUESTED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "docs-credit-memo-changing-logo"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Kamil! :1st_place_medal:

**Pull request merged**